spec/integration/api/v1/ingredients_spec.rb

```diff
@@ -33,7 +34,62 @@
     get 'Search ingredients' do
       consumes 'application/json'
       produces 'application/json'
-      description 'Retrieves ingredients by different providers'
+      description 'Retrieves ingredients from Jumbo'
+
+      response '200', 'ingredients retrieved' do
+        schema(
+          type: "object",
+          properties: {
+            data: {
+              type: "array",
+              items: { "$ref" => "#/definitions/provider_ingredient" }
+            }
+          },
+          required: [
+            :data
+          ]
+        )
+
+        run_test!
+      end
+
+      response '401', 'user unauthorized' do
+        let(:user_token) { 'invalid' }
+
+        run_test!
+      end
+    end
+  end
+
+  path '/search-lider-ingredients' do
+    let(:lider_client) { LiderClient.new }
+
+    parameter name: :user_email, in: :query, type: :string
+    parameter name: :user_token, in: :query, type: :string
+    parameter name: :query, in: :query, type: :string
+
+    let(:products) do
+      [
+        { price: "$ 1.150", measure: "500 g", name: "Pan coliza peruana", provider: "Lider" },
+        { price: "$ 770", measure: "500 g", name: "Pan hallulla delgada", provider: "Lider" },
+        { price: "$ 770", measure: "500 g", name: "Pan hallulla", provider: "Lider" },
+        { price: "$ 920", measure: "500 g", name: "Pan de hot dog", provider: "Lider" },
+        { price: "$ 995", measure: "500 g", name: "Pan doblada", provider: "Lider" }
+      ]
+    end
+    let(:query) { 'pan' }
+
+    before do
```

Comment:
**[rubocop]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>RSpec/ScatteredSetup: Do not define multiple before hooks in the same example group (also defined on lines 27, 128).

Reply:
los specs en swagger cree que hay muchos `before` blocks pero no los hay :monkey: